pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -621,6 +630,28 @@ func makeHttpHealthCheck(name, path string, port int32) *compute.HttpHealthCheck
 	}
 }
 
+// applyHttpHealthCheckChanges applies user configurations to the new HC while
+// discard the changes that are not applicable to the HC. It returns whether
+// the HTTP HC is drifted and needs update.
+func applyHttpHealthCheckChanges(hc, newHC *compute.HttpHealthCheck) (drift bool) {
+	if hc.CheckIntervalSec > newHC.CheckIntervalSec {
+		newHC.CheckIntervalSec = hc.CheckIntervalSec
```

Comment:
changed = true

Reply:
Here the hc.CheckIntervalSec > newHC.CheckIntervalSec means user sets a larger interval than our default interval.
This line of code is applying the user configured value to expected HC.
A comment will be added to clarify this piece of logic.